subsys/net/lib/openthread/platform/alarm.c

```diff
@@ -52,7 +52,14 @@ void platformAlarmProcess(otInstance *aInstance)
 {
 	if (timer_ms_fired) {
 		timer_ms_fired = false;
-		otPlatAlarmMilliFired(aInstance);
+#if defined(CONFIG_OPENTHREAD_DIAG)
+		if (otPlatDiagModeGet()) {
+			otPlatDiagAlarmFired(aInstance);
```

Comment:
It seems that `#include <openthread/platform/diag.h>` is missing, see failing CI.

Reply:
Fixed.